lms/templates/js/discussion/discussion-home.underscore

```diff
@@ -0,0 +1,59 @@
+<div class="discussion-article blank-slate">
+  <section class="home-header">
+    <span class="label"><%- gettext("DISCUSSION HOME:") %></span>
+    <% if (window.course_name) { %>
+      <h1 class="home-title"><%- window.course_name %></h1>
+    <% } %>
+  </section>
+
+  <% if (window.ENABLE_DISCUSSION_HOME_PANEL) { %>
+    <span class="label label-settings">
+      <%- gettext("How to use {platform_name} discussions").replace("{platform_name}", window.PLATFORM_NAME) %>
```

Comment:
I'm using `replace` instead of `interpolate` because we prefer `str.format` to `%` interpolation in Python, and I figured that the format would carry over to Javascript, to make things more consistent for our translators. We should probably standardize on this one way or the other: translators shouldn't have to know or care whether a string is written in Python or Javascript.

Also, I've now set `window.PLATFORM_NAME` in the `_underscore_template.html` template.